deploy/src/main/resources/production-marathon/environment.sh

```diff
@@ -3,6 +3,9 @@
 # The port that the CAF Audit Web Service is configured to listen on.
 export CAF_AUDIT_SERVICE_PORT=
 
+# The protocol used to connect with elasticsearch server.
```

Comment:
The protocol used to connect **to the Elasticsearch server**.

Reply:
Updated description.